Index: src/drivers/net/wireless/realtek/rtw89/Makefile
===================================================================
--- src.orig/drivers/net/wireless/realtek/rtw89/Makefile	2022-01-26 22:56:38.152946163 +0100
+++ src/drivers/net/wireless/realtek/rtw89/Makefile	2022-02-01 03:35:33.804917507 +0100
@@ -1,5 +1,9 @@
 # SPDX-License-Identifier: GPL-2.0 OR BSD-3-Clause
 
+CONFIG_RTW89_CORE := m
+CONFIG_RTW89_PCI := m
+CONFIG_RTW89_8852AE := m
+
 obj-$(CONFIG_RTW89_CORE) += rtw89_core.o
 rtw89_core-y += core.o \
 		mac80211.o \
